src/nifgen/metadata/functions_addon.py

```diff
@@ -57,7 +57,7 @@
     '.+Complex.+':                          { 'codegen_method': 'no',       },
     'GetStreamEndpointHandle':              { 'codegen_method': 'no',       },
     'AdjustSampleClockRelativeDelay':       { 'codegen_method': 'no',       },  # This is used internally by NI-TClk, but not by end users.
-	'.etAttributeViInt64':                  { 'codegen_method': 'no',       },  # Not used by python APIs
+    '.etAttributeViInt64':                  { 'codegen_method': 'no',       },  # Not used by python APIs
```

Comment:
Comment is inaccurate.
Should be something like "NI-FGEN has no ViInt64 attributes".

Reply:
Fixed in master.